pkgs/misc/emulators/desmume/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl
+, pkgconfig, libtool, intltool
+, libXmu
+, lua
+, agg, alsaLib, soundtouch, openal
+, desktop_file_utils
+, gtk2, gtkglext, libglade, pangox_compat
+, mesa, mesa_glu, libpcap, SDL, zziplib }:
+
+with stdenv.lib;
```

Comment:
As far as I can see you use stuff from stdenv.lib only in meta, thus it makes sense to put those into scope only there.